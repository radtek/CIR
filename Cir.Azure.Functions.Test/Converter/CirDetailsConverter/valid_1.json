{
  "cirSubmissionData": {
    "cirId": 123,
    "id": "id_1",
    "state": "state_1",
    "imageReferences": [
      {
        "imageId": "imageId_1",
        "url": "url_1"
      }
    ],
    "data": {
      "txtTurbineNumber": 44,
      "txtLocalturbineId": "txtLocalturbineId_1",
      "ddlBladeLength": "8",
      "txtBladeSerialNumber": "txtBladeSerialNumber_1",
      "dtCommissioningdate": "dtCommissioningdate_1",
      "dtInspectionDate": "dtInspectionDate_1",
      "txtMKversion": "txtMKversion_1",
      "txtSBURecommendation": "txtSBURecommendation_1",
      "imagecomponentKey": {
        "uploadedImagesCache": [
          {
            "imageId": "imageId_1",
            "damageSeverity": 2,
            "damageType":  "damageType_1"
          }
        ],
        "sections": {
          "section4": {
            "side": "side_1",
            "images": [
              {
                "imageId": "imageId_1",
                "number": 1
              }
            ]
          }
        }
      }
    }
  }
}