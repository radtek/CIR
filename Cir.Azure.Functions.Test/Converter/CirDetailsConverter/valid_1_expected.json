{
  "data": [
    {
      "turbineId": 44,
      "cirGuid": "id_1",
      "cirId": 123,
      "localTurbineId": "txtLocalturbineId_1",

      "inspectionDate": "dtInspectionDate_1",
      "state": "state_1",
      "commissioningDate": "dtCommissioningdate_1",
      "bladeSerialNumber": "txtBladeSerialNumber_1",
      "bladeLength": "AL23,8",
      "mkVersion": "txtMKversion_1",
      "sbuRecommendation": "txtSBURecommendation_1",
      "section4": {
        "side": "side_1",
        "images": [
          {
            "imageGuid": "imageId_1",
            "imageNumber": 1,
            "url": "url_1",
            "severity": 2,
            "damageType": "damageType_1",
            "damageDescription": null
          }
        ]
      },
      "noSection": {
        "overallBladeCondition": null,
        "images": []
      },
      "plateImageUrl": null
    }
  ]
}